backend/app/utils.py

```
@@ -115,58 +119,54 @@
     expire = datetime.utcnow() + timedelta(minutes=15)  # Token expires in 15 min
     data = {"sub": email, "exp": expire}
     return jwt.encode(data, SECRET_KEY, algorithm=ALGORITHM)
-    
-
 
 
 def create_access_token(data: dict, expires_delta: timedelta = None):
     """Create a JWT access token."""
     to_encode = data.copy()
-    expire = datetime.utcnow() + (expires_delta or timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES))
+    expire = datetime.utcnow() + (
```

Ruff (0.8.2)

`datetime.datetime.utcnow()` used

(DTZ003)